openedx/features/caliper_tracking/settings.py

```diff
@@ -0,0 +1,17 @@
+"""
+All settings for Caliper Tracking app.
+"""
+
+LOGIN_EVENT_EMITTER = 'caliper_tracking.helpers.emit_login_event'
+CALIPER_TRACKING_PROCESSOR = 'openedx.features.caliper_tracking.processor.CaliperProcessor'
+
+CALIPER_TRACKING_APP_NAME = 'openedx.features.caliper_tracking'
```

Comment:
Shouldn't the name be only `caliper_tracking`?

Reply:
App wasn't loading with only `caliper_tracking`.

Reply:
Test and confirm the installation method from PyPI (python site-packages).